lib/screens/chat_screens/user_chats.dart

```diff
@@ -0,0 +1,153 @@
+        if (!snapshot.hasData) {
+          return Center(
+            child: CircularProgressIndicator(),
+          );
+        }
+        final conversations = snapshot.data.docs;
+        List<ConversationBubble> conversationBubbles = [];
+        for (var userChat in conversations) {
+          final conversationText = userChat.data()['text'];
+          final conversationStarter = userChat.data()['chatSender'];
+          final conversationUserID = userChat.data()['chatIDUser'];
+          final recipeintID = userChat.data()['recipientId'];
+          final recipientName = userChat.data()['recipient'];
+          final conversationBubble = ConversationBubble(
+              recipient: recipeintID == loggedInUser.uid
+                  ? conversationStarter
+                  : recipientName,
+              recipientId: recipeintID == loggedInUser.uid
+                  ? conversationUserID
+                  : recipeintID,
+              text: conversationText);
+          if (conversationUserID == loggedInUser.uid ||
+              recipeintID == loggedInUser.uid) {
+            conversationBubbles.add(conversationBubble);
+          }
+        }
+        return Expanded(
+          child: ListView(
+            padding: EdgeInsets.symmetric(horizontal: 10.0, vertical: 20.0),
+            children: conversationBubbles,
+          ),
+        );
+      },
+    );
+  }
+}
+
+class ConversationBubble extends StatelessWidget {
+  ConversationBubble(
+      {this.recipient, this.text, this.recipientId, this.photoUrl});
+  final String recipient;
+  final String text;
+  final String recipientId;
+  final String photoUrl;
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(10.0),
+      child: Container(
+        padding: const EdgeInsets.all(10.0),
+        decoration: BoxDecoration(
+            color: Colors.cyan,
+            shape: BoxShape.rectangle,
+            borderRadius: BorderRadius.all(Radius.circular(25.0))),
+        child: FlatButton(
+          onPressed: () {
+            Navigator.push(
+                context,
+                MaterialPageRoute(
+                    builder: (context) => ChatScreen(recipientId, recipient)));
+          },
+          child: Row(
+            children: <Widget>[
+              Material(
+                child: Icon(
+                  Icons.account_circle,
+                  size: 50.0,
+                  color: Colors.grey,
+                ),
+                borderRadius: BorderRadius.all(Radius.circular(25.0)),
+                clipBehavior: Clip.hardEdge,
+              ),
+              Flexible(
+                child: Column(
+                  children: <Widget>[
+                    Container(
+                      child: Text(
+                          'This Question title is very long. I dont know why its long. It should not be this long. its some fucking bullshit', //Qusetion Title from backend should go here
```

Comment:
Any ideas on how the Question title is going to be added to this field? If it seems to be too complex for now, I would suggest changing the chatbox to only have the user's First and Last name and not focus on it right now.

Reply:
I agree with Max here.